InventoryManagement/src/test/java/tim6/inventorymanagement/article/integration/CreateArticleTest.java

```diff
@@ -0,0 +1,457 @@
+package tim6.inventorymanagement.article.integration;
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.test.context.SpringBootTest;
+import org.springframework.boot.test.web.client.TestRestTemplate;
+import org.springframework.boot.web.server.LocalServerPort;
+import org.springframework.http.*;
+import org.springframework.test.context.jdbc.Sql;
+import org.springframework.test.context.junit4.SpringRunner;
+import org.springframework.web.util.UriComponentsBuilder;
+import tim6.inventorymanagement.container.AbstractContainerBaseTest;
+import tim6.inventorymanagement.util.AuthUtil;
+import tim6.inventorymanagement.web.advice.ValidationErrorResponse;
+import tim6.inventorymanagement.web.advice.Violation;
+import tim6.inventorymanagement.web.v1.dto.article.ArticleGetDTO;
+import tim6.inventorymanagement.web.v1.dto.article.ArticlePostDTO;
+
+import java.math.BigDecimal;
+import java.net.URI;
+import java.util.List;
+import java.util.UUID;
+
+import static org.junit.Assert.*;
+import static tim6.inventorymanagement.util.ArticleUtil.assertArticleContentsEqual;
+
+@RunWith(SpringRunner.class)
+@SpringBootTest(webEnvironment = SpringBootTest.WebEnvironment.RANDOM_PORT)
+@Sql(scripts = "/articleTestData.sql", executionPhase = Sql.ExecutionPhase.BEFORE_TEST_METHOD)
+public class CreateArticleTest extends AbstractContainerBaseTest {
+
+    private final String ADMIN_USERNAME = "admin";
+    private final String ADMIN_PASSWORD = "admin";
+
+    private final String VALID_ARTICLE_NAME = "testArticle";
+    private final String VALID_ARTICLE_DESCRIPTION = "testDescription";
+    private final BigDecimal VALID_ARTICLE_PRICE = BigDecimal.valueOf(100);
+    private final BigDecimal VALID_ARTICLE_PRICE_ZERO = BigDecimal.ZERO;
+    private final BigDecimal INVALID_ARTICLE_PRICE_NEGATIVE = BigDecimal.valueOf(-100);
+    private final int VALID_ARTICLE_STOCK = 100;
+    private final int VALID_ARTICLE_STOCK_ZERO = 0;
+    private final int INVALID_ARTICLE_STOCK_NEGATIVE = -100;
+    private final String VALID_ARTICLE_TEST_IMAGE_URL = "testImageUrl";
+    private final String DEFAULT_ARTICLE_IMAGE_URL =
+            "https://res.cloudinary.com/dtddfx5ww/image/upload/v1621075575/WebStore/placeholder.jpg";
+
+    @Autowired private TestRestTemplate testRestTemplate;
+    @LocalServerPort private int port;
+
+    @Test
+    public void testCreateArticleReturnsCreated() throws Exception {
```

Comment:
There is no need to separate status code and body assertions in multiple tests. I suggest you combine them and remove unnecessary tests where you assert response body presents.
Same combining can be done for bad request tests.